translations/en-us.yml

```diff
@@ -1905,6 +1905,7 @@ osf-components:
         uploading_file: 'Uploading {fileCount, plural, one {# file} other {# files}}'
         upload_failed: '{fileCount, plural, one {# file} other {# files}} failed, click to try again'
         upload_success: '{fileCount, plural, one {# file} other {# files}} uploaded successfully'
+        exceeds_storage_limit: '- Exceeds storage limit'
```

Comment:
If https://github.com/CenterForOpenScience/ember-osf-web/pull/1552 gets merged in first, could you use its `error_storage: 'Insufficient storage at target location'` instead of this?